README.md

```diff
@@ -67,29 +73,29 @@ protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
     });
 }
 ```
-Please find example in the SimplePositiveTests class in the test project
-
-## New in 4.0.0
-
-Since the 4.0.0 version the FileBaseContext supports persisting data in the CSV files.
-The CSV files are stored in the directory with the database name. 
-The CSV files are named by the entity name. 
-The first row in the CSV file is the header with the column names.
+Please find an example in the SimplePositiveTests class in the test project
 
-## ! Braking changes in 3.0.0 !
+## Changes in 3.0.0
 
-In 3.0.0 version the provider was changed to support numeric values without quotation marks.
+In version 3.0.0 the provider was changed to support numeric values without quotation marks.
 
-```
+```json
 {
     "IntProp": 42,
     "LongProperty": 420,
     "DateTime": "2023-12-26T19:28:08"
 }
 ```
 
-The led to breaking changes in the provider. If you have used the provider before, you need to manualy update the database files. 
-The changes also affect on DateTime and DateTimeOffset values. The values are stored as string in the database.
-First run of the application could be slow becasuse a lot of System.Text.Json.JsonException will be provided.
-Performance be fixed after provider saves a database to files. While that the data will be stored in new formats.
-If you still have performance issues you need to manualy update the database files.
+This led to **breaking changes** in the provider. If you have used the provider before you'll need to manually update the database files. 
+The changes also affect `DateTime` and `DateTimeOffset` values as those values are stored as a string in the database file.
+The first run of the application could be slow because multiple `System.Text.Json.JsonException` will be generated.
+Performance will improve after the provider saves the database to files as the data will then be stored in the new format.
+If you still have performance issues you will need to manually update the database files.
+
+## New in 4.0.0
+
+Since version 4.0.0 the provider supports persisting data in CSV files.
+The CSV files are stored in the directory using the database name. 
+The CSV files are named using the entity name. 
+The first row in the CSV file is the header row with the column names.
```

Comment:
It would be good to have the order of news from newest to oldest version.
As a user, I want to know the information about the latest version instead of scrolling down.

And please add info for version 5.x